index.js

```diff
@@ -126,6 +127,7 @@ module.exports = {
         'react/jsx-key': 2,
         'react/jsx-no-comment-textnodes': 2,
         'react/jsx-no-duplicate-props': 2,
+        'react/jsx-no-namespace': 2,
```

Comment:
adding anything to the recommended config is a breaking change.
```suggestion
```

Reply:
I'll remove it then.